<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameOfLifeClass</class>
 <widget class="QWidget" name="GameOfLifeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GameOfLife</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelView">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditView"/>
     </item>
     <item>
      <widget class="QLabel" name="labelScene">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditScene"/>
     </item>
     <item>
      <widget class="QLabel" name="labelAdditional">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditAdditional"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="GraphicsView" name="graphicsView">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>GraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="GameOfLife.qrc"/>
 </resources>
 <connections/>
</ui>
